### ISSUE TITLE
Clarification on Basic Bank Account Data scope

### ISSUE THREAD

**Issue by jakubvozarik** [ConsumerDataStandardsAustralia/standards-maintenance#331]
Hi,

Basic Bank Account Data scope documentation says that it does not include account identifiers: 

Basic Bank Account Data | bank:accounts.basic:read | This scope would allow for the third party to access basic information of the customer’s accounts.Includes simple account information including balance.**Does not include account identifiers**, product information or transaction data.
-- | -- | --

However GET /banking/accounts documentation specifies that accountId is a mandatory field and bank:accounts.basic:read is required to perform this operation.

I would like to understand what the expectation for a DH when an ADR calls GET /banking/accounts with bank:accounts.basic:read with regards to accountId field in the response?

Thank you

**Comment by perlboy** [comment]
@jakubvozarik the `accountId` is a pairwise identifier as per the doc `A unique ID of the account adhering to the standards for ID permanence`. The account identifiers here refers to the bank account number (`accountNumber` in BankingAccountDetail).

**Comment by jakubvozarik** [comment]
thanks @perlboy for the lightning fast response! :)
 The term "account identifiers" feels a bit ambiguous here so thought I'd double check

**Comment by perlboy** [comment]
> thanks @perlboy for the lightning fast response! :)
> The term "account identifiers" feels a bit ambiguous here so thought I'd double check

Yes, perhaps it is, might be worth a minor change ticket @CDR-API-Stream

**Comment by CDR-API-Stream** [comment]
Thanks @perlboy and @jakubvozarik 

This text can be clarified in the next release of the standards. Propose to clarify as:

> Does not include detailed account information such as account numbers, product information or transaction data.

**Comment by CDR-API-Stream** [comment]
This change is staged for review: https://github.com/ConsumerDataStandardsAustralia/standards-staging/compare/release/1.11.1...maintenance/331

The change is a documentation fix to correctly clarify the basic account data scope excludes account numbers not account identifiers (which have mixed meanings and may be interpreted as the accountId subject to ID Permanence rules). This change is intended for release v1.11.1.
